File: 1erdepartamental/c++/8.promedio/main.cpp
#include <iostream>
#include "core/Estadisticas.h"

int main() {
    Estadisticas e(10, 25, 5, 30, 15);
    
    std::cout << "Suma: " << e.suma() << std::endl;
    std::cout << "Promedio: " << e.promedio() << std::endl;
    std::cout << "Máximo: " << e.maximo() << std::endl;
    std::cout << "Mínimo: " << e.minimo() << std::endl;
    
    return 0;
}

File: 1erdepartamental/python/8.promedio/main.py
"""
Programa: Promedio
Demuestra estadísticas con parámetros fijos
"""
#clase principal
from core.estadisticas import Estadisticas

#funcion principal 
def main ():
    e = Estadisticas(10, 25, 5, 30, 15)

    print(f"Suma: {e.suma()}")
    print(f"Promedio: {e.promedio()}")
    print(f"Máximo: {e.maximo()}")
    print(f"Mínimo: {e.minimo()}")
 
#Funcion que inicia todo el codigo
if __name__ == "__main__":
    main ()
